docs/docs/09-faqs.md

```diff
@@ -53,3 +53,8 @@ This tells the LLM (by way of a [system message](https://platform.openai.com/doc
 
 This context also automatically shares the `sys.ls`, `sys.read`, and `sys.write` tools with the tool that is using it as a context. This is because if a tool intends to interact with the workspace, it minimally needs these tools.
 
+### What do I need to get GPTScript working on windows?
```

Comment:
Windows should be capitalized

Reply:
Also, I'm not sure this FAQ question is quite specific enough - a lot of people will already have these tools installed for them. You might want to make the question reflect the specific issue the vcredist install solves a little more.